util/StringUtil.cc

```diff
@@ -19,6 +19,7 @@
 #include <limits>
 #include <cctype>
 #include <cstdio>
+#include <cstdlib>
```

Comment:
Please explain what function this meant to include.

Reply:
Added.